command-reference.erb

```diff
@@ -10,31 +10,28 @@ next: /rubygems-org-api
 
 This reference was automatically generated from RubyGems version <%= rubygems_version %>.
 
-<% commands.each do |name, command| %>* [gem <%= name %>](#gem-<%= name %>)
-<% end %>
+<%- commands.each do |name, command| -%><%= "* [gem #{name}](#gem-#{name})\n" %><%- end -%>
+<%- commands.each do |name, command| -%>
 
-<% commands.each do |name, command| %>
 ## gem <%= name %>
 
 <%= htmlify(command.summary) %>
 
-### Usage
+### Usage:
```

Comment:
I would rather remove the colon everywhere than add it everywhere. But that's just a matter of taste.

Reply:
I agree it doesn't add much, updated!